Admin.js

```diff
@@ -114,561 +141,775 @@ var Admin = {
             Admin.stopEvent(event);
 
-            var container = jQuery(this).closest('[data-prototype]');
-            var counter = ++Admin.lolCounters[container.attr('id')];
-            var proto = container.attr('data-prototype');
-            var protoName = container.attr('data-prototype-name') || '__name__';
+            const container = jQuery(this).closest("[data-prototype]");
+            const counter = ++Admin.lolCounters[container.attr("id")];
+            let proto = container.attr("data-prototype");
+            const protoName = container.attr("data-prototype-name") || "__name__";
             // Set field id
-            var idRegexp = new RegExp(container.attr('id')+'_'+protoName,'g');
-            proto = proto.replace(idRegexp, container.attr('id')+'_'+counter);
+            const idRegexp = new RegExp(`${container.attr("id")}_${protoName}`, "g");
+            proto = proto.replace(idRegexp, `${container.attr("id")}_${counter}`);
 
             // Set field name
-            var parts = container.attr('id').split('_');
-            var nameRegexp = new RegExp(parts[parts.length-1]+'\\]\\['+protoName,'g');
-            proto = proto.replace(nameRegexp, parts[parts.length-1]+']['+counter);
-            jQuery(proto)
-                .insertBefore(jQuery(this).parent())
-                .trigger('sonata-admin-append-form-element')
-            ;
-
-            jQuery(this).trigger('sonata-collection-item-added');
+            const parts = container.attr("id").split("_");
+            const nameRegexp = new RegExp(`${parts[parts.length - 1]}\\]\\[${protoName}`, "g");
+            proto = proto.replace(nameRegexp, `${parts[parts.length - 1]}][${counter}`);
+            jQuery(proto).
+                insertBefore(jQuery(this).parent()).
+                trigger("sonata-admin-append-form-element");
+            jQuery(this).trigger("sonata-collection-item-added");
+
         });
 
-        jQuery(subject).on('click', '.sonata-collection-delete', function(event) {
+        jQuery(subject).on("click", ".sonata-collection-delete", function (event) {
+
             Admin.stopEvent(event);
 
-            jQuery(this).trigger('sonata-collection-item-deleted');
+            jQuery(this).trigger("sonata-collection-item-deleted");
 
-            jQuery(this).closest('.sonata-collection-row').remove();
+            jQuery(this).closest(".sonata-collection-row").
+                remove();
+
+            jQuery(document).trigger("sonata-collection-item-deleted-successful");
 
-            jQuery(document).trigger('sonata-collection-item-deleted-successful');
         });
+
     },
 
-    setup_per_page_switcher: function(subject) {
-        Admin.log('[core|setup_per_page_switcher] setup page switcher', subject);
+    "setup_per_page_switcher" (subject) {
+
+        Admin.log("[core|setup_per_page_switcher] setup page switcher", subject);
+
+        jQuery("select.per-page").change(function (event) {
```

Comment:
- eslint: Unexpected unnamed function.
- eslint: 'event' is defined but never used.